<?xml version='1.0' encoding='utf-8'?>
<testsuites tests="147" failures="25" errors="0" skipped="45" time="0.0"><testsuite name="SoftwareManagement.Plugin.apama_plugin_install" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_install
13:21:29 INFO  Dir:   plugin_apama/
13:21:29 INFO  Title: Validate apama plugin install
13:21:29 INFO  ==============================================================
13:21:29 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.06 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_mon_support" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_mon_support
13:21:29 INFO  Dir:   plugin_apama/
13:21:29 INFO  Title: Validate mon file support of apama plugin
13:21:29 INFO  ==============================================================
13:21:29 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.02 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_remove
13:21:29 INFO  Dir:   plugin_apama/
13:21:29 INFO  Title: Validate apama plugin remove
13:21:29 INFO  ==============================================================
13:21:29 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.05 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_update" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
13:21:29 INFO  Dir:   plugin_apama/
13:21:29 INFO  Title: Validate apama plugin updating one project with another
13:21:29 INFO  ==============================================================
13:21:29 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.02 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>14:12:01 INFO  ==============================================================
14:12:01 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
14:12:01 INFO  Dir:   plugin_apama/
14:12:01 INFO  Title: Validate apama plugin updating one project with another
14:12:01 INFO  ==============================================================
14:12:01 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
14:12:01 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
14:12:02 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
14:12:02 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
14:12:03 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
14:12:04 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
14:12:04 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
14:12:05 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
14:12:05 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
14:12:05 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
14:12:05 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

14:12:05 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
14:12:05 WARN  Aborted test due to BLOCKED outcome
14:12:05 INFO  
14:12:05 INFO  cleanup:
14:12:05 INFO  Removing apama project
14:12:21 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
14:12:21 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
14:12:22 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
14:12:22 INFO  
14:12:22 INFO  Test duration: 21.42 secs
14:12:22 INFO  Test final outcome:  BLOCKED
14:12:22 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
14:12:22 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>14:15:21 INFO  ==============================================================
14:15:21 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
14:15:21 INFO  Dir:   plugin_apama/
14:15:21 INFO  Title: Validate apama plugin updating one project with another
14:15:21 INFO  ==============================================================
14:15:21 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
14:15:22 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
14:15:22 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
14:15:23 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
14:15:24 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
14:15:25 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
14:15:25 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
14:15:25 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
14:15:25 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
14:15:26 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
14:15:26 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

14:15:26 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
14:15:26 WARN  Aborted test due to BLOCKED outcome
14:15:26 INFO  
14:15:26 INFO  cleanup:
14:15:26 INFO  Removing apama project
14:15:42 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
14:15:42 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
14:15:43 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
14:15:43 INFO  
14:15:43 INFO  Test duration: 21.37 secs
14:15:43 INFO  Test final outcome:  BLOCKED
14:15:43 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
14:15:43 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_download_path" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:08 INFO  ==============================================================
13:31:08 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
13:31:08 INFO  Dir:   software_management_end_to_end/
13:31:08 INFO  Title: Install package to directory specified in tedge config set tmp.path
13:31:08 INFO  ==============================================================
13:31:08 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:08 WARN  Aborted test due to SKIPPED outcome
13:31:08 INFO  
13:31:08 INFO  Test duration: 0.06 secs
13:31:08 INFO  Test final outcome:  SKIPPED
13:31:08 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:08 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>13:54:08 INFO  ==============================================================
13:54:08 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
13:54:08 INFO  Dir:   software_management_end_to_end/
13:54:08 INFO  Title: Install package to directory specified in tedge config set tmp.path
13:54:08 INFO  ==============================================================
13:54:08 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
13:54:08 INFO  EnvironmentC8y Setup
13:54:09 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
13:54:24 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:54:25 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
13:54:26 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
13:54:27 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
13:54:29 INFO  State of current operation: SUCCESSFUL
13:54:30 INFO  Assert that (False == value) with value=False ... passed
13:54:30 INFO  Executed sudo, exit status 0
13:54:31 INFO  Executed sudo, exit status 0
13:54:31 INFO  Executed sudo, exit status 0
13:54:35 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:54:35 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
13:54:35 INFO  Delaying execution by 7 seconds
13:54:56 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:54:57 INFO  Response status: 201
13:54:57 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-05-20T13:54:57.097Z",
        "status": "PENDING"
    },
    "creationTime": "2022-05-20T13:54:57.079Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19422037",
    "id": "19422037",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
13:54:57 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-05-20T13:54:57.097Z', 'status': 'PENDING'}, 'creationTime': '2022-05-20T13:54:57.079Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19422037', 'id': '19422037', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
13:54:57 WARN  , retrying in 0.5 seconds...
13:54:57 WARN  , retrying in 0.5 seconds...
13:54:58 WARN  , retrying in 0.5 seconds...
13:54:58 WARN  , retrying in 0.5 seconds...
13:54:59 WARN  , retrying in 0.5 seconds...
13:54:59 WARN  , retrying in 0.5 seconds...
13:55:00 WARN  , retrying in 0.5 seconds...
13:55:00 WARN  , retrying in 0.5 seconds...
13:55:01 WARN  , retrying in 0.5 seconds...
13:55:01 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
13:55:02 WARN  AssertionError: ... blocked 
13:55:02 INFO  
13:55:02 INFO  cleanup:
13:55:02 INFO  Executed sudo, exit status 0
13:55:03 INFO  Executed sudo, exit status 0
13:55:07 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:55:07 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
13:55:07 INFO  Delaying execution by 7 seconds
13:55:29 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:55:33 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:55:33 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
13:55:33 INFO  
13:55:33 INFO  Test duration: 85.08 secs
13:55:33 INFO  Test final outcome:  BLOCKED
13:55:33 INFO  Test outcome reason: AssertionError:
13:55:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>14:15:43 INFO  ==============================================================
14:15:43 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
14:15:43 INFO  Dir:   software_management_end_to_end/
14:15:43 INFO  Title: Install package to directory specified in tedge config set tmp.path
14:15:43 INFO  ==============================================================
14:15:44 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:15:44 INFO  EnvironmentC8y Setup
14:15:45 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:15:59 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:15:59 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:16:00 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:16:00 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:16:03 INFO  State of current operation: FAILED
14:16:03 INFO  Assert that (False == value) with value=False ... passed
14:16:03 INFO  Executed sudo, exit status 0
14:16:04 INFO  Executed sudo, exit status 0
14:16:05 INFO  Executed sudo, exit status 0
14:16:09 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:16:09 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
14:16:09 INFO  Delaying execution by 8 seconds
14:16:30 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
14:16:31 INFO  Response status: 201
14:16:31 INFO  Response to action: {
    "creationTime": "2022-05-20T14:16:31.199Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426199",
    "id": "19426199",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
14:16:31 INFO  Started operation: {'creationTime': '2022-05-20T14:16:31.199Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426199', 'id': '19426199', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
14:16:31 WARN  , retrying in 0.5 seconds...
14:16:31 WARN  , retrying in 0.5 seconds...
14:16:32 WARN  , retrying in 0.5 seconds...
14:16:32 WARN  , retrying in 0.5 seconds...
14:16:33 WARN  , retrying in 0.5 seconds...
14:16:33 WARN  , retrying in 0.5 seconds...
14:16:34 WARN  , retrying in 0.5 seconds...
14:16:34 WARN  , retrying in 0.5 seconds...
14:16:35 WARN  , retrying in 0.5 seconds...
14:16:35 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
14:16:35 WARN  AssertionError: ... blocked 
14:16:35 INFO  
14:16:35 INFO  cleanup:
14:16:36 INFO  Executed sudo, exit status 0
14:16:36 INFO  Executed sudo, exit status 0
14:16:41 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:16:41 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
14:16:41 INFO  Delaying execution by 8 seconds
14:17:03 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:17:08 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:17:08 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:17:08 INFO  
14:17:08 INFO  Test duration: 85.37 secs
14:17:08 INFO  Test final outcome:  BLOCKED
14:17:08 INFO  Test outcome reason: AssertionError:
14:17:08 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_fail" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_fail
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for a failing installation
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the apt plugin for installation and removal of packages
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_default" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove_default
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the default plugin for installation and removal of packages
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True" type="FAILED" />
		<system-out>14:02:23 INFO  ==============================================================
14:02:23 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
14:02:23 INFO  Dir:   software_management_end_to_end/
14:02:23 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
14:02:23 INFO  ==============================================================
14:02:23 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:02:23 INFO  EnvironmentC8y Setup
14:02:23 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:02:23 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
14:02:23 INFO  Delaying execution by 6 seconds
14:02:43 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:02:44 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:02:44 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:02:45 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:02:47 INFO  State of current operation: SUCCESSFUL
14:02:47 INFO  Assert that (False == value) with value=False ... passed
14:02:48 INFO  Assert that (False == value) with value=False ... passed
14:02:48 INFO  Assert that (False == value) with value=False ... passed
14:02:48 INFO  Package robotfindskitten is installed
14:02:48 WARN  Assert that (False == value) with value=True ... failed 
14:02:48 INFO  Response status: 201
14:02:48 INFO  Response to action: {
    "creationTime": "2022-05-20T14:02:48.705Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426101",
    "id": "19426101",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:02:48 INFO  Started operation: {'creationTime': '2022-05-20T14:02:48.705Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426101', 'id': '19426101', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
14:02:50 INFO  State of operation 19426101 : PENDING
14:02:53 INFO  State of operation 19426101 : EXECUTING
14:02:55 INFO  State of operation 19426101 : EXECUTING
14:02:57 INFO  State of operation 19426101 : EXECUTING
14:02:59 INFO  State of operation 19426101 : EXECUTING
14:03:02 INFO  State of operation 19426101 : EXECUTING
14:03:04 INFO  State of operation 19426101 : EXECUTING
14:03:07 INFO  State of operation 19426101 : EXECUTING
14:03:10 INFO  State of operation 19426101 : EXECUTING
14:03:12 INFO  State of operation 19426101 : EXECUTING
14:03:14 INFO  State of operation 19426101 : EXECUTING
14:03:16 INFO  State of operation 19426101 : EXECUTING
14:03:19 INFO  State of operation 19426101 : EXECUTING
14:03:21 INFO  State of operation 19426101 : EXECUTING
14:03:23 INFO  State of operation 19426101 : EXECUTING
14:03:25 INFO  State of operation 19426101 : EXECUTING
14:03:28 INFO  State of operation 19426101 : EXECUTING
14:03:30 INFO  State of operation 19426101 : EXECUTING
14:03:32 INFO  State of operation 19426101 : EXECUTING
14:03:35 INFO  State of operation 19426101 : EXECUTING
14:03:37 INFO  State of operation 19426101 : EXECUTING
14:03:40 INFO  State of operation 19426101 : EXECUTING
14:03:42 INFO  State of operation 19426101 : EXECUTING
14:03:45 INFO  State of operation 19426101 : EXECUTING
14:03:47 INFO  State of operation 19426101 : EXECUTING
14:03:50 INFO  State of operation 19426101 : EXECUTING
14:03:52 INFO  State of operation 19426101 : EXECUTING
14:03:54 INFO  State of operation 19426101 : EXECUTING
14:03:56 INFO  State of operation 19426101 : EXECUTING
14:03:59 INFO  State of operation 19426101 : EXECUTING
14:04:01 INFO  State of operation 19426101 : EXECUTING
14:04:03 INFO  State of operation 19426101 : EXECUTING
14:04:06 INFO  State of operation 19426101 : EXECUTING
14:04:08 INFO  State of operation 19426101 : SUCCESSFUL
14:04:08 INFO  Assert that (False == value) with value=False ... passed
14:04:08 INFO  Package asciijump is installed
14:04:08 INFO  Assert that (True == value) with value=True ... passed
14:04:09 INFO  Package squirrel3 is installed
14:04:09 INFO  Assert that (True == value) with value=True ... passed
14:04:09 INFO  Package robotfindskitten is installed
14:04:09 INFO  Assert that (True == value) with value=True ... passed
14:04:09 INFO  Response status: 201
14:04:09 INFO  Response to action: {
    "creationTime": "2022-05-20T14:04:09.538Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426111",
    "id": "19426111",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:04:09 INFO  Started operation: {'creationTime': '2022-05-20T14:04:09.538Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426111', 'id': '19426111', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
14:04:11 INFO  State of operation 19426111 : PENDING
14:04:14 INFO  State of operation 19426111 : EXECUTING
14:04:16 INFO  State of operation 19426111 : EXECUTING
14:04:18 INFO  State of operation 19426111 : EXECUTING
14:04:20 INFO  State of operation 19426111 : EXECUTING
14:04:22 INFO  State of operation 19426111 : EXECUTING
14:04:25 INFO  State of operation 19426111 : EXECUTING
14:04:28 INFO  State of operation 19426111 : EXECUTING
14:04:30 INFO  State of operation 19426111 : EXECUTING
14:04:33 INFO  State of operation 19426111 : EXECUTING
14:04:35 INFO  State of operation 19426111 : EXECUTING
14:04:37 INFO  State of operation 19426111 : EXECUTING
14:04:40 INFO  State of operation 19426111 : EXECUTING
14:04:42 INFO  State of operation 19426111 : EXECUTING
14:04:44 INFO  State of operation 19426111 : EXECUTING
14:04:47 INFO  State of operation 19426111 : EXECUTING
14:04:49 INFO  State of operation 19426111 : EXECUTING
14:04:52 INFO  State of operation 19426111 : EXECUTING
14:04:54 INFO  State of operation 19426111 : EXECUTING
14:04:57 INFO  State of operation 19426111 : EXECUTING
14:04:59 INFO  State of operation 19426111 : EXECUTING
14:05:01 INFO  State of operation 19426111 : EXECUTING
14:05:03 INFO  State of operation 19426111 : EXECUTING
14:05:05 INFO  State of operation 19426111 : EXECUTING
14:05:08 INFO  State of operation 19426111 : EXECUTING
14:05:11 INFO  State of operation 19426111 : EXECUTING
14:05:13 INFO  State of operation 19426111 : EXECUTING
14:05:15 INFO  State of operation 19426111 : SUCCESSFUL
14:05:15 INFO  Assert that (False == value) with value=False ... passed
14:05:15 INFO  Assert that (False == value) with value=False ... passed
14:05:16 INFO  Assert that (False == value) with value=False ... passed
14:05:16 INFO  Assert that (False == value) with value=False ... passed
14:05:16 INFO  
14:05:16 INFO  cleanup:
14:05:20 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:05:21 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:05:21 INFO  
14:05:21 INFO  Test duration: 177.96 secs
14:05:21 INFO  Test final outcome:  FAILED
14:05:21 INFO  Test outcome reason: Assert that (False == value) with value=True
14:05:21 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True (+2 other failures)" type="FAILED" />
		<system-out>14:23:31 INFO  ==============================================================
14:23:31 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
14:23:31 INFO  Dir:   software_management_end_to_end/
14:23:31 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
14:23:31 INFO  ==============================================================
14:23:31 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:23:31 INFO  EnvironmentC8y Setup
14:23:31 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:23:32 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
14:23:32 INFO  Delaying execution by 7 seconds
14:23:53 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:23:53 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:23:54 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:23:54 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:23:56 INFO  State of current operation: SUCCESSFUL
14:23:57 INFO  Assert that (False == value) with value=False ... passed
14:23:57 INFO  Package asciijump is installed
14:23:57 WARN  Assert that (False == value) with value=True ... failed 
14:23:57 INFO  Package squirrel3 is installed
14:23:58 WARN  Assert that (False == value) with value=True ... failed 
14:23:58 INFO  Package robotfindskitten is installed
14:23:58 WARN  Assert that (False == value) with value=True ... failed 
14:23:58 INFO  Response status: 201
14:23:58 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-05-20T14:23:58.453Z",
        "status": "PENDING"
    },
    "creationTime": "2022-05-20T14:23:58.435Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426282",
    "id": "19426282",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:23:58 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-05-20T14:23:58.453Z', 'status': 'PENDING'}, 'creationTime': '2022-05-20T14:23:58.435Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426282', 'id': '19426282', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
14:24:00 INFO  State of operation 19426282 : PENDING
14:24:02 INFO  State of operation 19426282 : EXECUTING
14:24:05 INFO  State of operation 19426282 : EXECUTING
14:24:07 INFO  State of operation 19426282 : EXECUTING
14:24:09 INFO  State of operation 19426282 : EXECUTING
14:24:11 INFO  State of operation 19426282 : EXECUTING
14:24:14 INFO  State of operation 19426282 : EXECUTING
14:24:17 INFO  State of operation 19426282 : EXECUTING
14:24:19 INFO  State of operation 19426282 : EXECUTING
14:24:22 INFO  State of operation 19426282 : EXECUTING
14:24:24 INFO  State of operation 19426282 : EXECUTING
14:24:26 INFO  State of operation 19426282 : EXECUTING
14:24:28 INFO  State of operation 19426282 : EXECUTING
14:24:31 INFO  State of operation 19426282 : EXECUTING
14:24:33 INFO  State of operation 19426282 : EXECUTING
14:24:35 INFO  State of operation 19426282 : SUCCESSFUL
14:24:35 INFO  Assert that (False == value) with value=False ... passed
14:24:36 INFO  Package asciijump is installed
14:24:36 INFO  Assert that (True == value) with value=True ... passed
14:24:36 INFO  Package squirrel3 is installed
14:24:36 INFO  Assert that (True == value) with value=True ... passed
14:24:36 INFO  Package robotfindskitten is installed
14:24:36 INFO  Assert that (True == value) with value=True ... passed
14:24:36 INFO  Response status: 201
14:24:36 INFO  Response to action: {
    "delivery": {
        "log": [],
        "time": "2022-05-20T14:24:36.774Z",
        "status": "PENDING"
    },
    "creationTime": "2022-05-20T14:24:36.754Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426297",
    "id": "19426297",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:24:36 INFO  Started operation: {'delivery': {'log': [], 'time': '2022-05-20T14:24:36.774Z', 'status': 'PENDING'}, 'creationTime': '2022-05-20T14:24:36.754Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426297', 'id': '19426297', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
14:24:39 INFO  State of operation 19426297 : PENDING
14:24:41 INFO  State of operation 19426297 : EXECUTING
14:24:43 INFO  State of operation 19426297 : EXECUTING
14:24:45 INFO  State of operation 19426297 : EXECUTING
14:24:47 INFO  State of operation 19426297 : EXECUTING
14:24:50 INFO  State of operation 19426297 : EXECUTING
14:24:52 INFO  State of operation 19426297 : EXECUTING
14:24:55 INFO  State of operation 19426297 : EXECUTING
14:24:57 INFO  State of operation 19426297 : EXECUTING
14:24:59 INFO  State of operation 19426297 : EXECUTING
14:25:02 INFO  State of operation 19426297 : EXECUTING
14:25:04 INFO  State of operation 19426297 : EXECUTING
14:25:06 INFO  State of operation 19426297 : EXECUTING
14:25:08 INFO  State of operation 19426297 : EXECUTING
14:25:11 INFO  State of operation 19426297 : EXECUTING
14:25:13 INFO  State of operation 19426297 : EXECUTING
14:25:15 INFO  State of operation 19426297 : EXECUTING
14:25:18 INFO  State of operation 19426297 : EXECUTING
14:25:21 INFO  State of operation 19426297 : EXECUTING
14:25:23 INFO  State of operation 19426297 : EXECUTING
14:25:25 INFO  State of operation 19426297 : EXECUTING
14:25:27 INFO  State of operation 19426297 : EXECUTING
14:25:29 INFO  State of operation 19426297 : EXECUTING
14:25:32 INFO  State of operation 19426297 : EXECUTING
14:25:34 INFO  State of operation 19426297 : EXECUTING
14:25:37 INFO  State of operation 19426297 : EXECUTING
14:25:39 INFO  State of operation 19426297 : EXECUTING
14:25:41 INFO  State of operation 19426297 : EXECUTING
14:25:43 INFO  State of operation 19426297 : SUCCESSFUL
14:25:44 INFO  Assert that (False == value) with value=False ... passed
14:25:44 INFO  Assert that (False == value) with value=False ... passed
14:25:44 INFO  Assert that (False == value) with value=False ... passed
14:25:44 INFO  Assert that (False == value) with value=False ... passed
14:25:44 INFO  
14:25:44 INFO  cleanup:
14:25:48 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:25:49 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:25:49 INFO  
14:25:49 INFO  Test duration: 138.16 secs
14:25:49 INFO  Test final outcome:  FAILED
14:25:49 INFO  Test outcome reason: Assert that (False == value) with value=True (+2 other failures)
14:25:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple_mixed" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False" type="BLOCKED" />
		<system-out>14:05:21 INFO  ==============================================================
14:05:21 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
14:05:21 INFO  Dir:   software_management_end_to_end/
14:05:21 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
14:05:21 INFO  ==============================================================
14:05:21 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:05:21 INFO  EnvironmentC8y Setup
14:05:21 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:05:21 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
14:05:21 INFO  Delaying execution by 6 seconds
14:05:41 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
14:05:42 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:05:42 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:05:43 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:05:45 INFO  State of current operation: SUCCESSFUL
14:05:45 INFO  Assert that (False == value) with value=False ... passed
14:05:45 INFO  Assert that (False == value) with value=False ... passed
14:05:46 INFO  Assert that (False == value) with value=False ... passed
14:05:46 INFO  Assert that (False == value) with value=False ... passed
14:05:46 INFO  Assert that (False == value) with value=False ... passed
14:05:47 INFO  Response status: 201
14:05:47 INFO  Response to action: {
    "creationTime": "2022-05-20T14:05:47.849Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426126",
    "id": "19426126",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:05:47 INFO  Started operation: {'creationTime': '2022-05-20T14:05:47.849Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426126', 'id': '19426126', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
14:05:50 INFO  State of operation 19426126 : PENDING
14:05:52 INFO  State of operation 19426126 : EXECUTING
14:05:54 INFO  State of operation 19426126 : EXECUTING
14:05:56 INFO  State of operation 19426126 : EXECUTING
14:05:59 INFO  State of operation 19426126 : EXECUTING
14:06:01 INFO  State of operation 19426126 : EXECUTING
14:06:03 INFO  State of operation 19426126 : EXECUTING
14:06:06 INFO  State of operation 19426126 : EXECUTING
14:06:08 INFO  State of operation 19426126 : EXECUTING
14:06:11 INFO  State of operation 19426126 : EXECUTING
14:06:13 INFO  State of operation 19426126 : EXECUTING
14:06:15 INFO  State of operation 19426126 : EXECUTING
14:06:17 INFO  State of operation 19426126 : EXECUTING
14:06:20 INFO  State of operation 19426126 : EXECUTING
14:06:22 INFO  State of operation 19426126 : EXECUTING
14:06:24 INFO  State of operation 19426126 : EXECUTING
14:06:27 INFO  State of operation 19426126 : EXECUTING
14:06:29 INFO  State of operation 19426126 : EXECUTING
14:06:32 INFO  State of operation 19426126 : EXECUTING
14:06:34 INFO  State of operation 19426126 : EXECUTING
14:06:36 INFO  State of operation 19426126 : EXECUTING
14:06:39 INFO  State of operation 19426126 : EXECUTING
14:06:41 INFO  State of operation 19426126 : EXECUTING
14:06:44 INFO  State of operation 19426126 : EXECUTING
14:06:46 INFO  State of operation 19426126 : EXECUTING
14:06:48 INFO  State of operation 19426126 : EXECUTING
14:06:50 INFO  State of operation 19426126 : EXECUTING
14:06:53 INFO  State of operation 19426126 : EXECUTING
14:06:55 INFO  State of operation 19426126 : EXECUTING
14:06:57 INFO  State of operation 19426126 : EXECUTING
14:07:00 INFO  State of operation 19426126 : EXECUTING
14:07:02 INFO  State of operation 19426126 : EXECUTING
14:07:04 INFO  State of operation 19426126 : EXECUTING
14:07:06 INFO  State of operation 19426126 : EXECUTING
14:07:09 INFO  State of operation 19426126 : EXECUTING
14:07:11 INFO  State of operation 19426126 : EXECUTING
14:07:13 INFO  State of operation 19426126 : EXECUTING
14:07:15 INFO  State of operation 19426126 : EXECUTING
14:07:18 INFO  State of operation 19426126 : EXECUTING
14:07:21 INFO  State of operation 19426126 : EXECUTING
14:07:23 INFO  State of operation 19426126 : EXECUTING
14:07:25 INFO  State of operation 19426126 : EXECUTING
14:07:28 INFO  State of operation 19426126 : EXECUTING
14:07:30 INFO  State of operation 19426126 : EXECUTING
14:07:32 INFO  State of operation 19426126 : EXECUTING
14:07:34 INFO  State of operation 19426126 : EXECUTING
14:07:37 INFO  State of operation 19426126 : EXECUTING
14:07:39 INFO  State of operation 19426126 : EXECUTING
14:07:41 INFO  State of operation 19426126 : EXECUTING
14:07:43 INFO  State of operation 19426126 : EXECUTING
14:07:46 INFO  State of operation 19426126 : EXECUTING
14:07:48 INFO  State of operation 19426126 : EXECUTING
14:07:50 INFO  State of operation 19426126 : EXECUTING
14:07:53 INFO  State of operation 19426126 : EXECUTING
14:07:55 INFO  State of operation 19426126 : EXECUTING
14:07:58 INFO  State of operation 19426126 : EXECUTING
14:08:00 INFO  State of operation 19426126 : EXECUTING
14:08:02 INFO  State of operation 19426126 : EXECUTING
14:08:05 INFO  State of operation 19426126 : EXECUTING
14:08:07 INFO  State of operation 19426126 : EXECUTING
14:08:09 INFO  State of operation 19426126 : EXECUTING
14:08:11 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
14:08:11 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
14:08:11 INFO  
14:08:11 INFO  cleanup:
14:08:18 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:08:18 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:08:18 INFO  
14:08:18 INFO  Test duration: 177.86 secs
14:08:18 INFO  Test final outcome:  BLOCKED
14:08:18 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False
14:08:18 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)" type="BLOCKED" />
		<system-out>14:25:49 INFO  ==============================================================
14:25:49 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
14:25:49 INFO  Dir:   software_management_end_to_end/
14:25:49 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
14:25:49 INFO  ==============================================================
14:25:49 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:25:49 INFO  EnvironmentC8y Setup
14:25:50 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:25:50 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
14:25:50 INFO  Delaying execution by 7 seconds
14:26:11 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:26:11 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:26:12 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:26:12 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:26:15 INFO  State of current operation: SUCCESSFUL
14:26:15 INFO  Assert that (False == value) with value=False ... passed
14:26:15 INFO  Assert that (False == value) with value=False ... passed
14:26:15 INFO  Assert that (False == value) with value=False ... passed
14:26:16 INFO  Assert that (False == value) with value=False ... passed
14:26:16 INFO  Package moon-buggy is installed
14:26:16 WARN  Assert that (False == value) with value=True ... failed 
14:26:17 INFO  Response status: 201
14:26:17 INFO  Response to action: {
    "creationTime": "2022-05-20T14:26:17.539Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426316",
    "id": "19426316",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
14:26:17 INFO  Started operation: {'creationTime': '2022-05-20T14:26:17.539Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426316', 'id': '19426316', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
14:26:19 INFO  State of operation 19426316 : PENDING
14:26:22 INFO  State of operation 19426316 : EXECUTING
14:26:24 INFO  State of operation 19426316 : EXECUTING
14:26:26 INFO  State of operation 19426316 : EXECUTING
14:26:28 INFO  State of operation 19426316 : EXECUTING
14:26:30 INFO  State of operation 19426316 : EXECUTING
14:26:33 INFO  State of operation 19426316 : EXECUTING
14:26:36 INFO  State of operation 19426316 : EXECUTING
14:26:39 INFO  State of operation 19426316 : EXECUTING
14:26:41 INFO  State of operation 19426316 : EXECUTING
14:26:43 INFO  State of operation 19426316 : EXECUTING
14:26:45 INFO  State of operation 19426316 : EXECUTING
14:26:48 INFO  State of operation 19426316 : EXECUTING
14:26:50 INFO  State of operation 19426316 : EXECUTING
14:26:52 INFO  State of operation 19426316 : EXECUTING
14:26:54 INFO  State of operation 19426316 : EXECUTING
14:26:57 INFO  State of operation 19426316 : EXECUTING
14:27:00 INFO  State of operation 19426316 : EXECUTING
14:27:02 INFO  State of operation 19426316 : EXECUTING
14:27:04 INFO  State of operation 19426316 : EXECUTING
14:27:06 INFO  State of operation 19426316 : EXECUTING
14:27:09 INFO  State of operation 19426316 : EXECUTING
14:27:11 INFO  State of operation 19426316 : EXECUTING
14:27:14 INFO  State of operation 19426316 : EXECUTING
14:27:16 INFO  State of operation 19426316 : EXECUTING
14:27:19 INFO  State of operation 19426316 : EXECUTING
14:27:21 INFO  State of operation 19426316 : EXECUTING
14:27:23 INFO  State of operation 19426316 : EXECUTING
14:27:25 INFO  State of operation 19426316 : EXECUTING
14:27:28 INFO  State of operation 19426316 : EXECUTING
14:27:30 INFO  State of operation 19426316 : EXECUTING
14:27:32 INFO  State of operation 19426316 : EXECUTING
14:27:34 INFO  State of operation 19426316 : EXECUTING
14:27:37 INFO  State of operation 19426316 : EXECUTING
14:27:39 INFO  State of operation 19426316 : EXECUTING
14:27:41 INFO  State of operation 19426316 : EXECUTING
14:27:43 INFO  State of operation 19426316 : EXECUTING
14:27:46 INFO  State of operation 19426316 : EXECUTING
14:27:48 INFO  State of operation 19426316 : EXECUTING
14:27:51 INFO  State of operation 19426316 : EXECUTING
14:27:53 INFO  State of operation 19426316 : EXECUTING
14:27:56 INFO  State of operation 19426316 : EXECUTING
14:27:58 INFO  State of operation 19426316 : EXECUTING
14:28:00 INFO  State of operation 19426316 : EXECUTING
14:28:02 INFO  State of operation 19426316 : EXECUTING
14:28:05 INFO  State of operation 19426316 : EXECUTING
14:28:07 INFO  State of operation 19426316 : EXECUTING
14:28:09 INFO  State of operation 19426316 : EXECUTING
14:28:12 INFO  State of operation 19426316 : EXECUTING
14:28:14 INFO  State of operation 19426316 : EXECUTING
14:28:17 INFO  State of operation 19426316 : EXECUTING
14:28:19 INFO  State of operation 19426316 : EXECUTING
14:28:22 INFO  State of operation 19426316 : EXECUTING
14:28:24 INFO  State of operation 19426316 : EXECUTING
14:28:27 INFO  State of operation 19426316 : EXECUTING
14:28:29 INFO  State of operation 19426316 : EXECUTING
14:28:31 INFO  State of operation 19426316 : EXECUTING
14:28:34 INFO  State of operation 19426316 : EXECUTING
14:28:36 INFO  State of operation 19426316 : EXECUTING
14:28:39 INFO  State of operation 19426316 : EXECUTING
14:28:41 INFO  State of operation 19426316 : EXECUTING
14:28:43 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
14:28:43 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
14:28:43 INFO  
14:28:43 INFO  cleanup:
14:28:47 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:28:48 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:28:48 INFO  
14:28:48 INFO  Test duration: 178.67 secs
14:28:48 INFO  Test final outcome:  BLOCKED
14:28:48 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)
14:28:48 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_with_version" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_apt_install_remove_with_version
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the apt plugin for packages with version
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
</testsuite><testsuite name="SoftwareManagement.sm_docker_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.03 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>14:12:58 INFO  ==============================================================
14:12:58 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
14:12:58 INFO  Dir:   software_management_end_to_end/
14:12:58 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
14:12:58 INFO  ==============================================================
14:13:00 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:13:00 INFO  EnvironmentC8y Setup
14:13:00 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:13:14 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:13:15 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:13:16 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:13:16 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:13:18 INFO  State of current operation: SUCCESSFUL
14:13:19 INFO  Response status: 201
14:13:19 INFO  Response to action: {
    "creationTime": "2022-05-20T14:13:18.969Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426189",
    "id": "19426189",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
14:13:19 INFO  Started operation: {'creationTime': '2022-05-20T14:13:18.969Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426189', 'id': '19426189', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
14:13:21 INFO  State of operation 19426189 : PENDING
14:13:23 INFO  State of operation 19426189 : EXECUTING
14:13:25 INFO  State of operation 19426189 : EXECUTING
14:13:27 INFO  State of operation 19426189 : EXECUTING
14:13:30 INFO  State of operation 19426189 : FAILED
14:13:30 ERROR Stopping as the operation has failed
14:13:30 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
14:13:30 WARN  SystemError: The operation has failed ... blocked 
14:13:30 INFO  
14:13:30 INFO  cleanup:
14:13:34 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:13:34 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:13:34 INFO  
14:13:34 INFO  Test duration: 35.87 secs
14:13:34 INFO  Test final outcome:  BLOCKED
14:13:34 INFO  Test outcome reason: SystemError: The operation has failed
14:13:34 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>14:31:25 INFO  ==============================================================
14:31:25 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
14:31:25 INFO  Dir:   software_management_end_to_end/
14:31:25 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
14:31:25 INFO  ==============================================================
14:31:25 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:31:25 INFO  EnvironmentC8y Setup
14:31:26 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:31:26 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
14:31:26 INFO  Delaying execution by 6 seconds
14:31:46 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
14:31:46 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:31:47 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:31:47 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:31:49 INFO  State of current operation: SUCCESSFUL
14:31:50 INFO  Response status: 201
14:31:50 INFO  Response to action: {
    "creationTime": "2022-05-20T14:31:50.009Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426371",
    "id": "19426371",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
14:31:50 INFO  Started operation: {'creationTime': '2022-05-20T14:31:50.009Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426371', 'id': '19426371', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
14:31:52 INFO  State of operation 19426371 : PENDING
14:31:54 INFO  State of operation 19426371 : EXECUTING
14:31:56 INFO  State of operation 19426371 : EXECUTING
14:31:58 INFO  State of operation 19426371 : EXECUTING
14:32:01 INFO  State of operation 19426371 : FAILED
14:32:01 ERROR Stopping as the operation has failed
14:32:01 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
14:32:01 WARN  SystemError: The operation has failed ... blocked 
14:32:01 INFO  
14:32:01 INFO  cleanup:
14:32:05 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:32:05 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:32:05 INFO  
14:32:05 INFO  Test duration: 40.24 secs
14:32:05 INFO  Test final outcome:  BLOCKED
14:32:05 INFO  Test outcome reason: SystemError: The operation has failed
14:32:05 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_fake_install_remove
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the dummy plugin for installation and removal of packages
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove_multiple" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_fake_install_remove_multiple
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple" />
</testsuite><testsuite name="SoftwareManagement.sm_mapper_fail_and_reconnect" tests="2" errors="0" failures="0" skipped="2" time="0"><testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
13:31:09 INFO  ==============================================================
13:31:09 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.02 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>14:33:39 INFO  ==============================================================
14:33:39 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
14:33:39 INFO  Dir:   software_management_end_to_end/
14:33:39 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
14:33:39 INFO  ==============================================================
14:33:39 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
14:33:39 INFO  
14:33:39 INFO  Test duration: 0.02 secs
14:33:39 INFO  Test final outcome:  SKIPPED
14:33:39 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
14:33:39 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_mixed_install_remove_multiple" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>13:31:09 INFO  ==============================================================
13:31:09 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
13:31:09 INFO  Dir:   software_management_end_to_end/
13:31:09 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
13:31:09 INFO  ==============================================================
13:31:09 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
13:31:09 WARN  Aborted test due to SKIPPED outcome
13:31:09 INFO  
13:31:09 INFO  Test duration: 0.05 secs
13:31:09 INFO  Test final outcome:  SKIPPED
13:31:09 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
13:31:09 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed (+1 other failures)" type="BLOCKED" />
		<system-out>14:33:40 INFO  ==============================================================
14:33:40 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
14:33:40 INFO  Dir:   software_management_end_to_end/
14:33:40 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
14:33:40 INFO  ==============================================================
14:33:40 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
14:33:40 INFO  EnvironmentC8y Setup
14:33:40 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
14:33:40 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
14:33:40 INFO  Delaying execution by 6 seconds
14:34:00 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
14:34:01 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
14:34:01 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
14:34:02 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
14:34:04 INFO  State of current operation: SUCCESSFUL
14:34:04 INFO  Package apple is installed
14:34:04 INFO  Assert that (True == value) with value=True ... passed
14:34:04 INFO  Package banana is installed
14:34:04 INFO  Assert that (True == value) with value=True ... passed
14:34:05 INFO  Package cherry is installed
14:34:05 INFO  Assert that (True == value) with value=True ... passed
14:34:05 INFO  Package asciijump is installed
14:34:05 WARN  Assert that (False == value) with value=True ... failed 
14:34:05 INFO  Assert that (False == value) with value=False ... passed
14:34:05 INFO  Response status: 201
14:34:05 INFO  Response to action: {
    "creationTime": "2022-05-20T14:34:05.862Z",
    "deviceId": "19408350",
    "deviceName": "m32sd10b",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426440",
    "id": "19426440",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "apple",
            "action": "install",
            "id": "5495053",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "banana",
            "action": "install",
            "id": "5494888",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "cherry",
            "action": "install",
            "id": "5495382",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "::docker",
            "url": " "
        }
    ]
}
14:34:05 INFO  Started operation: {'creationTime': '2022-05-20T14:34:05.862Z', 'deviceId': '19408350', 'deviceName': 'm32sd10b', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19426440', 'id': '19426440', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'apple', 'action': 'install', 'id': '5495053', 'version': '::fruits', 'url': ' '}, {'name': 'banana', 'action': 'install', 'id': '5494888', 'version': '::fruits', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'cherry', 'action': 'install', 'id': '5495382', 'version': '::fruits', 'url': ' '}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '::docker', 'url': ' '}]}
14:34:08 INFO  State of operation 19426440 : PENDING
14:34:10 INFO  State of operation 19426440 : EXECUTING
14:34:12 INFO  State of operation 19426440 : EXECUTING
14:34:14 INFO  State of operation 19426440 : EXECUTING
14:34:17 INFO  State of operation 19426440 : EXECUTING
14:34:19 INFO  State of operation 19426440 : EXECUTING
14:34:21 INFO  State of operation 19426440 : EXECUTING
14:34:24 INFO  State of operation 19426440 : EXECUTING
14:34:27 INFO  State of operation 19426440 : EXECUTING
14:34:29 INFO  State of operation 19426440 : EXECUTING
14:34:31 INFO  State of operation 19426440 : EXECUTING
14:34:33 INFO  State of operation 19426440 : EXECUTING
14:34:36 INFO  State of operation 19426440 : EXECUTING
14:34:38 INFO  State of operation 19426440 : EXECUTING
14:34:40 INFO  State of operation 19426440 : EXECUTING
14:34:42 INFO  State of operation 19426440 : EXECUTING
14:34:45 INFO  State of operation 19426440 : FAILED
14:34:45 ERROR Stopping as the operation has failed
14:34:45 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mixed_install_remove_multiple/run.py", line 95, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
14:34:45 WARN  SystemError: The operation has failed ... blocked 
14:34:45 INFO  
14:34:45 INFO  cleanup:
14:34:49 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
14:34:49 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
14:34:49 INFO  
14:34:49 INFO  Test duration: 69.70 secs
14:34:49 INFO  Test final outcome:  BLOCKED
14:34:49 INFO  Test outcome reason: SystemError: The operation has failed (+1 other failures)
14:34:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="agent_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="AgentInitSession" name="agent_init_session" />
</testsuite><testsuite name="apt_fail_noarg" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFailNoArg" name="apt_fail_noarg">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    apt_fail_noarg
13:21:29 INFO  Dir:   plugin_apt/
13:21:29 INFO  Title: Validate apt plugin fetures(prepare, list, install, remove finalize) without arguments
13:21:29 INFO  ==============================================================
13:21:29 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.04 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFailNoArg" name="apt_fail_noarg" />
</testsuite><testsuite name="apt_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFinalize" name="apt_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    apt_finalize
13:21:29 INFO  Dir:   plugin_apt/
13:21:29 INFO  Title: Validate apt plugin finalize - dense version
13:21:29 INFO  ==============================================================
13:21:29 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.02 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFinalize" name="apt_finalize" />
</testsuite><testsuite name="apt_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallDense" name="apt_install">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    apt_install
13:21:29 INFO  Dir:   plugin_apt/
13:21:29 INFO  Title: Validate apt plugin prepare - dense version
13:21:29 INFO  ==============================================================
13:21:29 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.02 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallDense" name="apt_install" />
</testsuite><testsuite name="apt_install_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFail" name="apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    apt_install_fail
13:21:29 INFO  Dir:   plugin_apt/
13:21:29 INFO  Title: Validate apt plugin prepare - failing
13:21:29 INFO  ==============================================================
13:21:29 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.02 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFail" name="apt_install_fail" />
</testsuite><testsuite name="apt_install_local_file" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    apt_install_local_file
13:21:29 INFO  Dir:   plugin_apt/
13:21:29 INFO  Title: Validate apt plugin install from local file - SUCCESS case
13:21:29 INFO  ==============================================================
13:21:29 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.02 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file" />
</testsuite><testsuite name="apt_install_local_file_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    apt_install_local_file_fail
13:21:29 INFO  Dir:   plugin_apt/
13:21:29 INFO  Title: Validate apt plugin install from local file - FAIL case
13:21:29 INFO  ==============================================================
13:21:29 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.02 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail" />
</testsuite><testsuite name="apt_install_local_file_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:29 INFO  ==============================================================
13:21:29 INFO  Id:    apt_install_local_file_with_version
13:21:29 INFO  Dir:   plugin_apt/
13:21:29 INFO  Title: Validate apt plugin install from local file AND with a version - SUCCESS case
13:21:29 INFO  ==============================================================
13:21:29 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:29 WARN  Aborted test due to SKIPPED outcome
13:21:29 INFO  
13:21:29 INFO  Test duration: 0.02 secs
13:21:29 INFO  Test final outcome:  SKIPPED
13:21:29 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:29 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version" />
</testsuite><testsuite name="apt_install_local_file_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:30 INFO  ==============================================================
13:21:30 INFO  Id:    apt_install_local_file_with_version_fail
13:21:30 INFO  Dir:   plugin_apt/
13:21:30 INFO  Title: Validate apt plugin install from local file AND with a version - FAIL case
13:21:30 INFO  ==============================================================
13:21:30 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:30 WARN  Aborted test due to SKIPPED outcome
13:21:30 INFO  
13:21:30 INFO  Test duration: 0.02 secs
13:21:30 INFO  Test final outcome:  SKIPPED
13:21:30 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:30 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail" />
</testsuite><testsuite name="apt_list_format" tests="2" errors="0" failures="0" skipped="0" time="0"><testcase classname="AptPluginListFormat" name="apt_list_format" />
<testcase classname="AptPluginListFormat" name="apt_list_format" />
</testsuite><testsuite name="apt_prepare" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepare" name="apt_prepare">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_prepare
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin install - dense version
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepare" name="apt_prepare" />
</testsuite><testsuite name="apt_prepare_install_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_prepare_install_finalize
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin install use case
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize" />
</testsuite><testsuite name="apt_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveDense" name="apt_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_remove
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin prepare - dense version
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveDense" name="apt_remove" />
</testsuite><testsuite name="apt_remove_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveFails" name="apt_remove_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_remove_fail
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin prepare - failing
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveFails" name="apt_remove_fail" />
</testsuite><testsuite name="apt_sequence_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_sequence_install_with_version
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin install with version use case
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version" />
</testsuite><testsuite name="apt_sequence_install_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_sequence_install_with_version_fail
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin install with version fails use case
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail" />
</testsuite><testsuite name="apt_sequence_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_sequence_remove
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin prepare install remove finalize use case
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove" />
</testsuite><testsuite name="apt_sequence_remove_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_sequence_remove_with_version
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin remove with version use case
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version" />
</testsuite><testsuite name="apt_sequence_remove_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_sequence_remove_with_version_fail
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin remove with version fails when the version not found
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail" />
</testsuite><testsuite name="apt_update_list" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginUpdateList" name="apt_update_list">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_update_list
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin update-list command use case
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginUpdateList" name="apt_update_list" />
</testsuite><testsuite name="apt_wrong_option" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginWrongOption" name="apt_wrong_option">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    apt_wrong_option
13:21:33 INFO  Dir:   plugin_apt/
13:21:33 INFO  Title: Validate apt plugin install
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginWrongOption" name="apt_wrong_option" />
</testsuite><testsuite name="c8y_jwt_token_retrieval" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateJWTTokenRetrieval" name="c8y_jwt_token_retrieval" />
</testsuite><testsuite name="c8y_restart_bridge" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RestartBridge" name="c8y_restart_bridge" />
</testsuite><testsuite name="call_tedge_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-V" />
</testsuite><testsuite name="call_tedge_-h" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h" />
</testsuite><testsuite name="call_tedge_-h_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h_-V" />
</testsuite><testsuite name="call_tedge_config_help" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_help" />
</testsuite><testsuite name="call_tedge_config_list" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_list" />
</testsuite><testsuite name="custom_init_system" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="CustomInitSystem" name="custom_init_system" />
</testsuite><testsuite name="docker_plugin_finalize" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    docker_plugin_finalize
13:21:33 INFO  Dir:   plugin_docker/
13:21:33 INFO  Title: Validate docker plugin finalize command
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.05 secs
13:21:33 INFO  Test final outcome:  SKIPPED
13:21:33 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:33 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<failure message="BLOCKED: sudo&lt;docker_pull&gt; returned non-zero exit code 1" type="BLOCKED" />
		<system-out>14:12:25 INFO  ==============================================================
14:12:25 INFO  Id:    docker_plugin_finalize
14:12:25 INFO  Dir:   plugin_docker/
14:12:25 INFO  Title: Validate docker plugin finalize command
14:12:25 INFO  ==============================================================
14:12:29 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:29 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out ... passed
14:12:31 WARN  Executed sudo&lt;docker_pull&gt;, exit status 1
14:12:31 INFO  Contents of docker_pull.err: 
  Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  -----

14:12:31 WARN  sudo&lt;docker_pull&gt; returned non-zero exit code 1 ... blocked 
14:12:31 WARN  Aborted test due to BLOCKED outcome
14:12:31 INFO  
14:12:31 INFO  cleanup:
14:12:31 INFO  
14:12:31 INFO  Test duration: 6.27 secs
14:12:31 INFO  Test final outcome:  BLOCKED
14:12:31 INFO  Test outcome reason: sudo&lt;docker_pull&gt; returned non-zero exit code 1
14:12:31 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_install" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:33 INFO  ==============================================================
13:21:33 INFO  Id:    docker_plugin_install
13:21:33 INFO  Dir:   plugin_docker/
13:21:33 INFO  Title: Validate docker plugin install
13:21:33 INFO  ==============================================================
13:21:33 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:33 WARN  Aborted test due to SKIPPED outcome
13:21:33 INFO  
13:21:33 INFO  Test duration: 0.02 secs
13:21:34 INFO  Test final outcome:  SKIPPED
13:21:34 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:34 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>14:12:32 INFO  ==============================================================
14:12:32 INFO  Id:    docker_plugin_install
14:12:32 INFO  Dir:   plugin_docker/
14:12:32 INFO  Title: Validate docker plugin install
14:12:32 INFO  ==============================================================
14:12:32 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:32 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_images.out ... passed
14:12:35 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
14:12:35 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:23]
14:12:36 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
14:12:36 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
14:12:36 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_ps.out contains "registry" ... failed [environment_docker_plugin.py:49,run.py:26]
14:12:36 INFO  
14:12:36 INFO  cleanup:
14:12:36 INFO  Removing containers scheduled to be cleaned up: ['']
14:12:37 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:12:37 INFO  Removing images scheduled to be cleaned up: ['registry']
14:12:38 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:12:38 INFO  
14:12:38 INFO  Test duration: 6.37 secs
14:12:38 INFO  Test final outcome:  BLOCKED
14:12:38 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
14:12:38 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_install_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:34 INFO  ==============================================================
13:21:34 INFO  Id:    docker_plugin_install_with_version
13:21:34 INFO  Dir:   plugin_docker/
13:21:34 INFO  Title: Validate docker plugin install
13:21:34 INFO  ==============================================================
13:21:34 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:34 WARN  Aborted test due to SKIPPED outcome
13:21:34 INFO  
13:21:34 INFO  Test duration: 0.02 secs
13:21:34 INFO  Test final outcome:  SKIPPED
13:21:34 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:34 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>14:12:38 INFO  ==============================================================
14:12:38 INFO  Id:    docker_plugin_install_with_version
14:12:38 INFO  Dir:   plugin_docker/
14:12:38 INFO  Title: Validate docker plugin install
14:12:38 INFO  ==============================================================
14:12:39 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:39 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.out ... passed
14:12:41 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
14:12:41 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:24]
14:12:42 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
14:12:43 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
14:12:43 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.out contains "registry:2.7.1" ... failed [environment_docker_plugin.py:49,run.py:28]
14:12:43 INFO  
14:12:43 INFO  cleanup:
14:12:43 INFO  Removing containers scheduled to be cleaned up: ['', '']
14:12:44 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:12:44 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
14:12:44 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:12:44 INFO  
14:12:44 INFO  Test duration: 6.52 secs
14:12:44 INFO  Test final outcome:  BLOCKED
14:12:44 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
14:12:44 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:34 INFO  ==============================================================
13:21:34 INFO  Id:    docker_plugin_remove
13:21:34 INFO  Dir:   plugin_docker/
13:21:34 INFO  Title: Validate docker plugin remove
13:21:34 INFO  ==============================================================
13:21:34 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:34 WARN  Aborted test due to SKIPPED outcome
13:21:34 INFO  
13:21:34 INFO  Test duration: 0.02 secs
13:21:34 INFO  Test final outcome:  SKIPPED
13:21:34 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:34 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>14:12:45 INFO  ==============================================================
14:12:45 INFO  Id:    docker_plugin_remove
14:12:45 INFO  Dir:   plugin_docker/
14:12:45 INFO  Title: Validate docker plugin remove
14:12:45 INFO  ==============================================================
14:12:45 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:45 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out ... passed
14:12:47 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
14:12:47 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:latest' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

14:12:47 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
14:12:47 WARN  Aborted test due to BLOCKED outcome
14:12:47 INFO  
14:12:47 INFO  cleanup:
14:12:47 INFO  Removing containers scheduled to be cleaned up: ['', '']
14:12:48 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:12:48 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
14:12:49 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:12:49 INFO  
14:12:49 INFO  Test duration: 4.55 secs
14:12:49 INFO  Test final outcome:  BLOCKED
14:12:49 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
14:12:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:34 INFO  ==============================================================
13:21:34 INFO  Id:    docker_plugin_remove_with_version
13:21:34 INFO  Dir:   plugin_docker/
13:21:34 INFO  Title: Validate docker plugin remove with version
13:21:34 INFO  ==============================================================
13:21:34 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:34 WARN  Aborted test due to SKIPPED outcome
13:21:34 INFO  
13:21:34 INFO  Test duration: 0.02 secs
13:21:34 INFO  Test final outcome:  SKIPPED
13:21:34 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:34 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>14:12:49 INFO  ==============================================================
14:12:49 INFO  Id:    docker_plugin_remove_with_version
14:12:49 INFO  Dir:   plugin_docker/
14:12:49 INFO  Title: Validate docker plugin remove with version
14:12:49 INFO  ==============================================================
14:12:50 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:50 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove_with_version/Output/linux/docker_images.out ... passed
14:12:52 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
14:12:52 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.7.1' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

14:12:52 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
14:12:52 WARN  Aborted test due to BLOCKED outcome
14:12:52 INFO  
14:12:52 INFO  cleanup:
14:12:52 INFO  Removing containers scheduled to be cleaned up: ['', '']
14:12:53 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:12:53 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
14:12:54 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:12:54 INFO  
14:12:54 INFO  Test duration: 4.55 secs
14:12:54 INFO  Test final outcome:  BLOCKED
14:12:54 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
14:12:54 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_update" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>13:21:34 INFO  ==============================================================
13:21:34 INFO  Id:    docker_plugin_update
13:21:34 INFO  Dir:   plugin_docker/
13:21:34 INFO  Title: Validate docker plugin install updates existing containers
13:21:34 INFO  ==============================================================
13:21:34 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
13:21:34 WARN  Aborted test due to SKIPPED outcome
13:21:34 INFO  
13:21:34 INFO  Test duration: 0.02 secs
13:21:34 INFO  Test final outcome:  SKIPPED
13:21:34 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
13:21:34 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>14:12:54 INFO  ==============================================================
14:12:54 INFO  Id:    docker_plugin_update
14:12:54 INFO  Dir:   plugin_docker/
14:12:54 INFO  Title: Validate docker plugin install updates existing containers
14:12:54 INFO  ==============================================================
14:12:55 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:12:55 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_images.out ... passed
14:12:57 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
14:12:57 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.6.2' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

14:12:57 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
14:12:57 WARN  Aborted test due to BLOCKED outcome
14:12:57 INFO  
14:12:57 INFO  cleanup:
14:12:57 INFO  Removing containers scheduled to be cleaned up: ['', '']
14:12:57 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
14:12:57 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry:2.7.1']
14:12:58 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
14:12:58 INFO  
14:12:58 INFO  Test duration: 4.55 secs
14:12:58 INFO  Test final outcome:  BLOCKED
14:12:58 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
14:12:58 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="invalid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="InvalidDeviceId" name="invalid_device_id" />
</testsuite><testsuite name="local_mqtt_pub_fail_noqos" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_noqos" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos0" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos0" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos1" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos1" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos2" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos2" />
</testsuite><testsuite name="local_mqtt_pub_sub" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub" />
</testsuite><testsuite name="local_mqtt_pub_sub_retain" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MqttPublishWithRetain" name="local_mqtt_pub_sub_retain">
		<failure message="FAILED: Grep on temp_sensor_sub.out contains &quot;\[test/alarms/temp_sensor\] alarm msg 2&quot; (+1 other failures)" type="FAILED" />
		<system-out>13:18:19 INFO  ==============================================================
13:18:19 INFO  Id:    local_mqtt_pub_sub_retain
13:18:19 INFO  Dir:   local_mqtt/
13:18:19 INFO  Title: Validate local publishing with retain flag
13:18:19 INFO  ==============================================================
13:18:20 INFO  Executed tedge&lt;pub_to_set_alarm&gt;, exit status 0
13:18:20 INFO  Executed tedge&lt;pub_to_set_alarm&gt;, exit status 0
13:18:21 INFO  Started tedge&lt;temp_sensor_sub&gt; with process id 24070
13:18:22 INFO  Executed tedge&lt;pub_to_clear_alarm&gt;, exit status 0
13:18:23 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
13:18:24 INFO  Executed tedge&lt;pub_to_clear_alarm&gt;, exit status 0
13:18:24 INFO  Started tedge&lt;temp_sensor_empty_message&gt; with process id 24075
13:18:25 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
13:18:25 WARN  Grep on temp_sensor_sub.out contains "\[test/alarms/temp_sensor\] alarm msg 2" ... failed [run.py:139]
13:18:26 WARN  Grep on temp_sensor_sub.out contains "\[test/alarms/temp_sensor\] " ... failed [run.py:142]
13:18:26 INFO  Grep on file temp_sensor_empty_message.out ... passed
13:18:26 INFO  
13:18:26 INFO  Test duration: 6.09 secs
13:18:26 INFO  Test final outcome:  FAILED
13:18:26 INFO  Test outcome reason: Grep on temp_sensor_sub.out contains "\[test/alarms/temp_sensor\] alarm msg 2" (+1 other failures)
13:18:26 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="local_mqtt_pub_sub_with_sudo" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub_with_sudo">
		<failure message="FAILED: Grep on tedge_sub.out contains &quot;amessage&quot;" type="FAILED" />
		<system-out>13:18:26 INFO  ==============================================================
13:18:26 INFO  Id:    local_mqtt_pub_sub_with_sudo
13:18:26 INFO  Dir:   local_mqtt/
13:18:26 INFO  Title: Validate local publishing and subscribing
13:18:26 INFO  ==============================================================
13:18:26 INFO  Started sudo&lt;tedge_sub&gt; with process id 24078
13:18:26 INFO  Executed sudo&lt;tedge_pub2&gt;, exit status 0
13:18:27 INFO  Executed sudo&lt;tedge_pub3&gt;, exit status 0
13:18:28 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
13:18:28 WARN  Grep on tedge_sub.out contains "amessage" ... failed [run.py:61]
13:18:28 INFO  Grep on file tedge_sub.out ... passed
13:18:28 INFO  
13:18:28 INFO  Test duration: 2.80 secs
13:18:28 INFO  Test final outcome:  FAILED
13:18:28 INFO  Test outcome reason: Grep on tedge_sub.out contains "amessage"
13:18:28 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="local_mqtt_sub_fail" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_sub_fail" />
</testsuite><testsuite name="log_request_error_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifySearchTextError" name="log_request_error_messages">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>13:18:30 INFO  ==============================================================
13:18:30 INFO  Id:    log_request_error_messages
13:18:30 INFO  Dir:   log_request_end_to_end/
13:18:30 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
13:18:30 INFO  ==============================================================
13:18:31 INFO  EnvironmentC8y Setup
13:18:32 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
13:18:32 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 2 seconds
13:18:32 INFO  Delaying execution by 9 seconds
13:18:55 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:18:56 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
13:18:57 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
13:18:57 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
13:18:58 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
13:18:58 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
13:18:59 INFO  log response None
13:18:59 WARN  retry, retrying in 1 seconds...
13:19:00 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/19406050/binaries
13:19:00 WARN  Assert that (True == value) with value=False ... failed [run.py:43]
13:19:00 INFO  
13:19:00 INFO  cleanup:
13:19:01 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
13:19:01 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
13:19:02 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
13:19:06 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:19:07 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
13:19:07 INFO  
13:19:07 INFO  Test duration: 36.39 secs
13:19:07 INFO  Test final outcome:  FAILED
13:19:07 INFO  Test outcome reason: Assert that (True == value) with value=False
13:19:07 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="log_request_num_lines" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifyNumberOfLines" name="log_request_num_lines">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>13:19:07 INFO  ==============================================================
13:19:07 INFO  Id:    log_request_num_lines
13:19:07 INFO  Dir:   log_request_end_to_end/
13:19:07 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
13:19:07 INFO  ==============================================================
13:19:07 INFO  EnvironmentC8y Setup
13:19:07 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
13:19:07 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
13:19:07 INFO  Delaying execution by 6 seconds
13:19:28 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:19:28 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
13:19:29 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
13:19:29 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
13:19:30 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
13:19:30 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
13:19:31 INFO  log response None
13:19:31 WARN  retry, retrying in 1 seconds...
13:19:32 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/19411151/binaries
13:19:32 INFO  nlines 0
13:19:32 WARN  Assert that (True == value) with value=False ... failed [run.py:44]
13:19:32 INFO  
13:19:32 INFO  cleanup:
13:19:33 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
13:19:33 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
13:19:34 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
13:19:38 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:19:39 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
13:19:39 INFO  
13:19:39 INFO  Test duration: 32.02 secs
13:19:39 INFO  Test final outcome:  FAILED
13:19:39 INFO  Test outcome reason: Assert that (True == value) with value=False
13:19:39 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mapper_awaits_before_reconnect" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED" />
		<system-out>13:34:13 INFO  ==============================================================
13:34:13 INFO  Id:    mapper_awaits_before_reconnect
13:34:13 INFO  Dir:   tedge_mapper_c8y/
13:34:13 INFO  Title: Validate mapper doesn't reconnect too often
13:34:13 INFO  ==============================================================
13:34:13 INFO  Started mosquitto_sub with process id 28763
13:34:13 INFO  Waiting for 0.1 seconds
13:34:13 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
13:34:13 INFO  Waiting for 5.0 seconds
13:34:19 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
13:34:20 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
13:34:20 WARN  Grep on tedge_mapper_journal.out contains "ERROR" ... failed [run.py:67]
13:34:20 INFO  Line count on file tedge_mapper_journal.out ... passed
13:34:20 WARN  Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0 ... failed [run.py:78]
13:34:20 WARN  Aborted test due to FAILED outcome
13:34:20 INFO  
13:34:20 INFO  Test duration: 7.58 secs
13:34:20 INFO  Test final outcome:  FAILED
13:34:20 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0
13:34:20 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_smaller_interval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringSmallInterval" name="monitoring_smaller_interval">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>13:21:09 INFO  ==============================================================
13:21:09 INFO  Id:    monitoring_smaller_interval
13:21:09 INFO  Dir:   monitoring/
13:21:09 INFO  Title: Validate the tedge dm agent messages that are produced with smaller sampling interval than the batching window. Here the simulated collectd messages are used
13:21:09 INFO  ==============================================================
13:21:10 INFO  Executed sudo&lt;collectd&gt;, exit status 0
13:21:10 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
13:21:10 INFO  Started sudo&lt;tedge_sub&gt; with process id 24882
13:21:11 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:12 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:12 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:13 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:13 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:14 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:15 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:15 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:16 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:16 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:17 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:17 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:18 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:18 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:19 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:20 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:20 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:21 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:21 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:22 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:23 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
13:21:23 INFO  0

13:21:23 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 104, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 114, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 134, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
13:21:23 WARN  TypeError: 'int' object is not subscriptable ... blocked 
13:21:23 INFO  
13:21:23 INFO  cleanup:
13:21:23 INFO  monitoring_cleanup
13:21:24 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
13:21:24 INFO  
13:21:24 INFO  Test duration: 15.04 secs
13:21:24 INFO  Test final outcome:  BLOCKED
13:21:24 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
13:21:24 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_collectd" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MonitoringWithCollectd" name="monitoring_with_collectd">
		<skipped message="SKIPPED: Test execution disabled due to #792 (instable)" />
		<system-out>13:21:24 INFO  ==============================================================
13:21:24 INFO  Id:    monitoring_with_collectd
13:21:24 INFO  Dir:   monitoring/
13:21:24 INFO  Title: Validate tedge dm agent messages that are produced by batching the collectd messages
13:21:24 INFO  ==============================================================
13:21:24 INFO  Test execution disabled due to #792 (instable) ... skipped
13:21:24 INFO  
13:21:24 INFO  Test duration: 0.03 secs
13:21:24 INFO  Test final outcome:  SKIPPED
13:21:24 INFO  Test outcome reason: Test execution disabled due to #792 (instable)
13:21:24 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_simulated_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringWithSimulatedMessages" name="monitoring_with_simulated_messages">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>13:21:24 INFO  ==============================================================
13:21:24 INFO  Id:    monitoring_with_simulated_messages
13:21:24 INFO  Dir:   monitoring/
13:21:24 INFO  Title: Validate the tedge dm agent messages that are produced from the simulated collectd messages
13:21:24 INFO  ==============================================================
13:21:25 INFO  Executed sudo&lt;collectd&gt;, exit status 0
13:21:25 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
13:21:25 INFO  Started sudo&lt;tedge_sub&gt; with process id 24950
13:21:26 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
13:21:27 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
13:21:28 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
13:21:28 INFO  0

13:21:28 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 98, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 108, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 127, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
13:21:28 WARN  TypeError: 'int' object is not subscriptable ... blocked 
13:21:28 INFO  
13:21:28 INFO  cleanup:
13:21:28 INFO  monitoring_cleanup
13:21:29 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
13:21:29 INFO  
13:21:29 INFO  Test duration: 4.49 secs
13:21:29 INFO  Test final outcome:  BLOCKED
13:21:29 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
13:21:29 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_change_connection_fails" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortChangeConnectionFails" name="mqtt_port_change_connection_fails" />
</testsuite><testsuite name="mqtt_port_change_connection_works" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MqttPortChangeConnectionWorks" name="mqtt_port_change_connection_works">
		<skipped message="SKIPPED: failing intermitently" />
		<system-out>13:20:14 INFO  ==============================================================
13:20:14 INFO  Id:    mqtt_port_change_connection_works
13:20:14 INFO  Dir:   misc_features/
13:20:14 INFO  Title: mqtt port change connection works for all the thin-edge services that use the mosquitto broker
13:20:14 INFO  ==============================================================
13:20:14 INFO  failing intermitently ... skipped
13:20:14 INFO  
13:20:14 INFO  Test duration: 0.02 secs
13:20:14 INFO  Test final outcome:  SKIPPED
13:20:14 INFO  Test outcome reason: failing intermitently
13:20:14 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_set" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortSet" name="mqtt_port_set" />
</testsuite><testsuite name="mqtt_port_unset" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortUnSet" name="mqtt_port_unset" />
</testsuite><testsuite name="publish_flux" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishFlux" name="publish_flux" />
</testsuite><testsuite name="publish_sawmill" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishSawmill" name="publish_sawmill" />
</testsuite><testsuite name="runtime_start_two_mappers" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RuntimeMultiMappers" name="runtime_start_two_mappers" />
</testsuite><testsuite name="sm_tedge_mapper_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeSMMapperInitSession" name="sm_tedge_mapper_init_session" />
</testsuite><testsuite name="smoketest_json" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmokeTestJson" name="smoketest_json" />
</testsuite><testsuite name="smoketest_json_400_samples_10ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples10ms" name="smoketest_json_400_samples_10ms" />
</testsuite><testsuite name="smoketest_json_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples20ms" name="smoketest_json_400_samples_20ms" />
</testsuite><testsuite name="smoketest_smartrest" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest" name="smoketest_smartrest" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_1Xms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples1Xms" name="smoketest_smartrest_400_samples_1Xms" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples20ms" name="smoketest_smartrest_400_samples_20ms" />
</testsuite><testsuite name="tedge_agent_user_sudo_access" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="TedgeAgentUserSudoAccess" name="tedge_agent_user_sudo_access">
		<skipped message="SKIPPED: #955 Observation: Exit code of the apt plugin when called with no arguments" />
		<system-out>13:31:35 INFO  ==============================================================
13:31:35 INFO  Id:    tedge_agent_user_sudo_access
13:31:35 INFO  Dir:   tedge/
13:31:35 INFO  Title: Validate tedge-agent user has a limited sudo right
13:31:35 INFO  ==============================================================
13:31:35 INFO  #955 Observation: Exit code of the apt plugin when called with no arguments ... skipped
13:31:35 INFO  
13:31:35 INFO  Test duration: 0.02 secs
13:31:35 INFO  Test final outcome:  SKIPPED
13:31:35 INFO  Test outcome reason: #955 Observation: Exit code of the apt plugin when called with no arguments
13:31:35 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_connect_test_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestNegative" name="tedge_connect_test_negative" />
</testsuite><testsuite name="tedge_connect_test_positive" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestPositive" name="tedge_connect_test_positive" />
</testsuite><testsuite name="tedge_connect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestSMServices" name="tedge_connect_test_sm_services" />
</testsuite><testsuite name="tedge_disconnect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeDisConnectTestSMServices" name="tedge_disconnect_test_sm_services" />
</testsuite><testsuite name="tedge_mapper_az_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzError" name="tedge_mapper_az_negative" />
</testsuite><testsuite name="tedge_mapper_az_with_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithTimestamp" name="tedge_mapper_az_with_timestamp" />
</testsuite><testsuite name="tedge_mapper_az_without_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithoutTimestamp" name="tedge_mapper_az_without_timestamp" />
</testsuite><testsuite name="tedge_mapper_c8y_alarms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yAlarm" name="tedge_mapper_c8y_alarms" />
</testsuite><testsuite name="tedge_mapper_c8y_child_device_measurement" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yChildDevice" name="tedge_mapper_c8y_child_device_measurement" />
</testsuite><testsuite name="tedge_mapper_c8y_packet_threshold_size" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yThresholdPacketSize" name="tedge_mapper_c8y_packet_threshold_size">
		<failure message="FAILED: Grep on tedge_sub.out contains &quot;The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184.&quot;" type="FAILED" />
		<system-out>13:35:57 INFO  ==============================================================
13:35:57 INFO  Id:    tedge_mapper_c8y_packet_threshold_size
13:35:57 INFO  Dir:   tedge_mapper_c8y/
13:35:57 INFO  Title: Validate the Cumulocity Mapper accepted threshold packet size when it receives a message on tedge/measurements
13:35:57 INFO  ==============================================================
13:35:57 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
13:35:57 INFO  Started sudo&lt;tedge_sub&gt; with process id 29354
13:35:58 INFO  Executed sudo&lt;tedge_msg&gt;, exit status 0
13:35:58 INFO  Executed sudo&lt;tedge_pub&gt;, exit status 0
13:35:59 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
13:35:59 WARN  Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184." ... failed [run.py:88]
13:35:59 INFO  
13:35:59 INFO  cleanup:
13:35:59 INFO  mapper_cleanup
13:36:00 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
13:36:00 INFO  
13:36:00 INFO  Test duration: 3.00 secs
13:36:00 INFO  Test final outcome:  FAILED
13:36:00 INFO  Test outcome reason: Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184."
13:36:00 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_software_update" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="tedge_software_update" />
</testsuite><testsuite name="valid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateValidDeviceId" name="valid_device_id" />
</testsuite></testsuites>